[2025-08-07 04:00:22,999] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_sla_dag.sleep_30 scheduled__2025-08-07T03:58:00+00:00 [queued]>
[2025-08-07 04:00:23,011] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_sla_dag.sleep_30 scheduled__2025-08-07T03:58:00+00:00 [queued]>
[2025-08-07 04:00:23,011] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2025-08-07 04:00:23,011] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2025-08-07 04:00:23,011] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2025-08-07 04:00:23,034] {taskinstance.py:1377} INFO - Executing <Task(_PythonDecoratedOperator): sleep_30> on 2025-08-07 03:58:00+00:00
[2025-08-07 04:00:23,039] {standard_task_runner.py:52} INFO - Started process 10801 to run task
[2025-08-07 04:00:23,043] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'example_sla_dag', 'sleep_30', 'scheduled__2025-08-07T03:58:00+00:00', '--job-id', '188', '--raw', '--subdir', '/home/***/.local/lib/python3.7/site-packages/***/example_dags/example_sla_dag.py', '--cfg-path', '/tmp/tmpyxg9291h', '--error-file', '/tmp/tmpvdh91kan']
[2025-08-07 04:00:23,045] {standard_task_runner.py:80} INFO - Job 188: Subtask sleep_30
[2025-08-07 04:00:23,115] {task_command.py:369} INFO - Running <TaskInstance: example_sla_dag.sleep_30 scheduled__2025-08-07T03:58:00+00:00 [running]> on host 92a46ac2453f
[2025-08-07 04:00:23,377] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=email@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=example_sla_dag
AIRFLOW_CTX_TASK_ID=sleep_30
AIRFLOW_CTX_EXECUTION_DATE=2025-08-07T03:58:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-08-07T03:58:00+00:00
[2025-08-07 07:58:32,886] {local_task_job.py:221} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2025-08-07 07:58:32,890] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 10801. PIDs of all processes in the group: [10801]
[2025-08-07 07:58:32,891] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 10801
[2025-08-07 07:58:32,891] {taskinstance.py:1541} ERROR - Received SIGTERM. Terminating subprocesses.
[2025-08-07 07:58:32,943] {process_utils.py:75} INFO - Process psutil.Process(pid=10801, status='terminated', exitcode=0, started='04:00:22') (10801) terminated with exit code 0
